presto-spi/src/main/java/com/facebook/presto/spi/MetadataUpdateRequest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+public interface MetadataUpdateRequest
```

Comment:
`ConnectorMetadataUpdateHandle`. We don't need to define anything in the field. Check how other handle interfaces are defined.

Reply:
We need to create update request object before sending to coordinator. To create that object we need `ConnectorMetadataUpdateHandle` to have basic interface functions right ?

Reply:
All these functions are called hive connector. We usually don't put anything in engine level. If we wanna delegate to underlying connectors, we do downcast within its connector.